backend/Spring Boot/springboot_starterproject/src/test/java/daiict/slop/springboot_starterproject/SpringbootStarterprojectApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package daiict.slop.springboot_starterproject;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class SpringbootStarterprojectApplicationTests {
+
+	@Test
```

Comment:
Any basic test is appreciated. Any simple addition test will also work. This will help the user to get some idea about how tests are written.